book/src/fundamentals/iterators.md

```diff
@@ -42,8 +43,11 @@ world.run(|u32s: View<u32>, usizes: View<usize>| {
 
 The iterator will only yield components from entities that have both `u32` and `usize` components, while ignoring the rest.
 
-You can use views in any order, but the same combination with the views in different positions might yield components in a different order. In general you shouldn't expect specific ordering from Shipyard's iterators.
+You can use views in any order.
+However, using the same combination of views in different positions might yield components in a different order.
+You shouldn't expect specific ordering from Shipyard's iterators, in general.
 
 ---
 
-Iteration is only one of the main assets of an ECS, in the next chapter we'll talk about perhaps the most important one: Systems.
+Fast iteration speed is only one of the main assets of an ECS.
```

Comment:
While it's fast, it'll depend on many things, it's probably best to just remove it from this chapter.
```suggestion
Iteration is only one of the main assets of an ECS.
```